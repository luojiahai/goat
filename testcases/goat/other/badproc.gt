proc main(val float c, val int d,val int e,)
    int a;
begin
    newstates := something [ab [cd, ef]];

end
