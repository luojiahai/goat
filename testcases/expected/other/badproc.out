Parse error at (line 1, column 44):
unexpected ")"
expecting "val" or "ref"
